orc8r/cloud/helm/orc8r/requirements.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The Magma Authors.
```

Comment:
For helm 3 requirements got moved to the Chart.yaml file so no need for this. You should see this if you rebase. Also, while you're at it can you update the `requirements.lock` -> `Chart.lock`. See: https://helm.sh/blog/helm-3-preview-pt5/)